hw/ip/entropy_src/doc/_index.md

```diff
@@ -10,34 +10,38 @@ This module conforms to the [Comportable guideline for peripheral functionality.
+  - Entropy bits are available for firmware consumption.
+  - The external analog entropy source is available.
+  - The external analog entropy source has bit polarity errors.
+  - The external analog entropy source has a FIFO error.
+  - An internal FIFO error has occurred.
+- Two health checks that defined by SP 800-90B are performed by this revision:  Repetition Count and Adaptive Proportion tests.
 - No alerts are supported by this revision.
 
 
 ## Description
 
 This IP block provides an entropy source that is part of a larger solution to generate true random numbers (a TRNG mechanism).
-The general solution for TRNG relies on firmware to orchestrate the majority of the function, getting helper services from specific hardware blocks.
+The general solution for TRNG relies on either hardware or firmware to implement the majority of the function, depending which functions are enabled by firmware or e-fuses.
 For purposes of reference in this document, the NIST term "noise" will be synonymous with the term "entropy".
 Likewise, no conditioning function will be performed in this block.
-For the generation of entropy, this block provides the entropy bits needed for the entropy source solution as required by the firmware.
+For the generation of entropy, this block provides the entropy bits needed for the entropy source solution as required by the RNG mechanism.
 
-At a high-level, this ENTROPY_SRC block, when enabled, will continuously collect entropy bits from the entropy source into a FIFO that can be read from the TLUL bus.
-For this revision, the entropy generation can be very fast, based on programming.
-Other sources could be much slower.
-The support logic in the block will support entropy sources of any rate.
+At a high-level, this ENTROPY_SRC block, when enabled, will continuously collect entropy bits from the entropy source into a FIFO that can be read from the TLUL bus, or sent out through a hardware interface.
+For this revision, the entropy generation, whether digital or analog, is by default tuned to generate at about the same rate. 
+The digital source can be programmed to be faster, or much slower if desired.
+A much slower rate of entropy might be useful in demonstrating how the downstream applications use the entropy.
 
-Once the required number of entropy bits have been collected, the firmware has two options of notification: an interrupt, or polling.
-For this revision, the `ENTROPY_RDY` bit will always be set if the {{< regref "ES_THRESH" >}} register is configured to a low value, so a firmware scheme of just reading the status bit to confirm that entropy is available is a good strategy.
-For slower sources, an interrupt may be a better use of CPU cycles.
+Once the threshold has been reached for collected entropy bits, the firmware has two options of notification: an interrupt, or polling.
+The `es_entropy_valid` bit will be set when the {{< regref "ES_THRESH" >}} register value has matched the value in the TLUL FIFO.
```

Comment:
just to clarify, shouldn't this be compared to the number of elements in the TLUL FIFO? E.g.:

The `es_entropy_valid` bit will be set when the number of elements in the TLUL FIFO is greater or equal the value in the {{< regref "ES_THRESH" >}} register.

Reply:
yes, that is clearer, phrasing changed.